internal/ansible/ansible_manager.go

```diff
@@ -61,6 +62,11 @@ func NewAnsibleManager(
 		return nil, fmt.Errorf("ansible manager cannot initialize mapping repository: %w", err)
 	}
 
+	_, err = exec.LookPath("ansible")
```

Comment:
maybe check two things here: 

- If Ansible version is supported. 
- If Ansible is executable for that user

Reply:
We can check ansible version but we are in control which version lands in the image.

Reply:
`ansible` should be installed via our blueprint. Since this is a "prototype", in my opinion, we can leave this check looser for the moment. WDYT?

Reply:
+1. Since we own the blueprint (at least for now), we know what it must contain.

Reply:
Works for me :+1: